src/bin/vip-whoami.ts

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env node
 
 /**
- * @flow
  * @format
  */
```

Comment:
I think we can drop the `@format` pragma as well.

In Prettier, `requirePragma` is `false` [by default](https://prettier.io/docs/en/options.html#require-pragma), and our [.prettierrc](https://github.com/Automattic/vip-cli/blob/4abd8cef3e807ceeb051d1bf49b080f75561c519/.prettierrc) does not speicify it either.